services/src/handlers/projects.rs

```diff
@@ -86,11 +100,72 @@ pub(crate) async fn list_projects_handler<C: ApplicationContext>(
     Ok(web::Json(listing))
 }
 
-/// Retrieves details about the latest version of a project.
+/// Updates a project.
+/// This will create a new version.
 #[utoipa::path(
     tag = "Projects",
-    get,
+    patch,
```

Comment:
We need changes to the UI, right?

Reply:
the API should not have changed but only be extended. The UI doesn't support versions anyway.

I think the diff git shows here is just confusing.